packages/system-data/src/fields/index.ts

```diff
@@ -1,58 +1,62 @@
 import defaults from './_defaults.yaml';
 
+import accessFields from './activity.yaml';
```

Comment:
That should probably be the following I assume:

```suggestion
import accessFields from './access.yaml';
```